### DISCUSSION TITLE
National Geographic Canada

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#741] [Q&A]
**Channel Name:** Nat Geo TV
**Channel ID:** NationalGeographic.ca
**Country:** Canada
**Language:** English
**Website:** https://www.natgeotv.com/ca/hd
**Stream URL (optional):**
**Notes (optional):** Be able to watch from Canada please.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19733
